packages/text-block/src/TextBlock.tsx

```diff
@@ -18,40 +17,30 @@ export const TextBlock: FC<BlockProps> = ({ appBridge }) => {
 
     const {
         columnNumber = DEFAULT_COLUMN_NUMBER,
-        content = [...Array(parseInt(columnNumber.toString()))],
+        content,
         isColumnGutterCustom,
         columnGutterCustom,
         columnGutterSimple,
     } = blockSettings;
 
-    const onTextChange = (value: string, index: number) => {
-        const newContent = [...content, (content[index] = value)];
-        setBlockSettings({ content: newContent });
-    };
-
+    const onTextChange = (value: string) => setBlockSettings({ content: value });
     return (
         <div
             data-test-id="text-block"
             style={{
                 gap: isColumnGutterCustom ? columnGutterCustom : columnGutterSimple,
             }}
-            className={`text-block tw-grid ${GRID_CLASSES[columnNumber]}`}
+            className={`tw-gap-2 tw-block ${hasRichTextValue(content) && GRID_CLASSES[columnNumber]}`}
```

Comment:
```suggestion
            className={`tw-gap-2 tw-block ${hasRichTextValue(content) ? GRID_CLASSES[columnNumber]  : ''}`}
```
otherwise it outputs `false` as a className